spec/dummy/config/environments/test.rb

```diff
@@ -14,7 +14,7 @@
 
   # Configure static asset server for tests with Cache-Control for performance.
   config.serve_static_assets  = true
-  config.static_cache_control = "public, max-age=3600"
+  config.static_cache_control = 'public, max-age=3600'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.